src/nupic/types/Sdr.cpp

```diff
@@ -426,6 +426,9 @@ namespace sdr {
         SDR::setDenseInplace();
     }
 
+    Reshape SparseDistributedRepresentation::flatten() const
+        { return Reshape(*this, {size} ); }
```

Comment:
do we want Reshape, or a plain SDR {size} + setSparse would do?

Reply:
That would work too, but it would copy the data.  The Reshape class does not copy data.